### PR TITLE
Add Patch104p keyword to Listening Outpost change comments

### PR DESCRIPTION
**Pull request by xezon**
Just updating comments.

### CODE REVIEW

File: Patch104pZH/GameFilesEdited/Data/INI/Object/NukeGeneral.ini [hunk 1]
```diff
@@ -4792,10 +4792,9 @@ Object Nuke_ChinaVehicleListeningOutpost
...
   End
 
-  ; @bugfix - hanfield
+  ; Patch104p @bugfix hanfield 22/08/2021
   ; Replaced the AssaultTransportAIUpdate module with TransportAIUpdate.
   ; This should make the Listening Outpost behave better in combat.
-  ; 22/08/2021
 
   ;Behavior = AssaultTransportAIUpdate ModuleTag_NewAI
   ;  MembersGetHealedAtLifeRatio = 0.5
```
Comment (xezon) [inline Patch104pZH/GameFilesEdited/Data/INI/Object/NukeGeneral.ini:4800]: I am curious about this setting. The new `TransportAIUpdate` that we added does not have the `MembersGetHealedAtLifeRatio = 0.5` setting. Does this mean the Outposts no longer heal passengers?

Comment (xezon) [reply Patch104pZH/GameFilesEdited/Data/INI/Object/NukeGeneral.ini:4800]: Seems `TransportAIUpdate` does not support `MembersGetHealedAtLifeRatio` setting.

Comment (MTKing4) [reply Patch104pZH/GameFilesEdited/Data/INI/Object/NukeGeneral.ini:4800]: Attack outposts don't heal passengers afaik, only when dismounted

Comment (xezon) [reply Patch104pZH/GameFilesEdited/Data/INI/Object/NukeGeneral.ini:4800]: Actually they do, same as Troopcrawler, when the infantry is boarded. I just tested.

**Review by commy2 — APPROVED** [review]

**Review by commy2 — APPROVED** [review]
Could also delete the commented out module. Nothing interesting to see.

**Pull request merged**